tests/functional/config.js

```diff
@@ -42,7 +42,7 @@ const sauceLabsConfig = {
 if (process.env.TRAVIS === 'true') {
   config = {
     ...config,
-    logLevel: 'debug',
+    logLevel: 'silent',
```

Comment:
Note that `silent` is the default logLevel for WebDriver.IO http://webdriver.io/guide/getstarted/configuration.html